test/e2e/regression/api_test.go

```diff
@@ -118,7 +118,7 @@ func (s *APIRegressionTestSuite) TestOrganizationAPI() {
 		userResp, err := s.testBench.Client.CreateUser(ctxOrgAdminAuth, &shieldv1beta1.CreateUserRequest{Body: &shieldv1beta1.UserRequestBody{
 			Name:  "acme 2 member",
 			Email: "acme-member@odpf.io",
-			Slug:  "acme-2-member",
+			Slug:  "acme_2_member",
```

Comment:
Apologies for noticing this late but we should also allow hyphens in slug. I think I'll merge this PR and you can raise another one with just this change and proton as well. Does that make sense?

Reply:
yes. Also one more change from your recent commit in main. Object namespace regex needs to have a `/`. Otherwise 
`shield/organisation` was failing according to the validator we have added in this PR.

Will add both of these changes in the next proto https://github.com/odpf/proton/pull/262/files

Reply:
Let me update the user slug regex, you can then simply fix this PR only.

Reply:
Cool, will merge this PR then